app/src/main/java/io/islnd/android/islnd/app/database/IslndProvider.java

```diff
@@ -557,6 +620,18 @@ public Uri insert(Uri uri, ContentValues values) {
                 returnUri = insertEventWithAliasAndUri(db, uri);
                 break;
             }
+            case NOTIFICATION: {
+                long _id = db.insertWithOnConflict(
```

Comment:
Currently the table create statement in `IslndDbHelper` does not have a unique constraint, so there will never be a conflict. I think we can use the regular insert here...